Frameworks/ImageIO/CGImageSource.mm

```diff
@@ -489,21 +1370,109 @@ size_t CGImageSourceGetCount(CGImageSourceRef isrc) {
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Caveat
+ @Notes A limited subset of source container properties are supported currently  
 */
 CFDictionaryRef CGImageSourceCopyProperties(CGImageSourceRef isrc, CFDictionaryRef options) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    RETURN_NULL_IF(!isrc);
+    NSData* imageData = ((ImageSource*)isrc).data;
+    RETURN_NULL_IF(!imageData);
+
+    NSMutableDictionary *properties = [[NSMutableDictionary alloc] init]; 
```

Comment:
I did but it didn't detect anything. I may have set it up incorrectly so it wasn't diffing against the correct master and thought there were no changes.